src/toast/utils.py

```diff
@@ -36,6 +36,64 @@
 )
 
 
+def set_numba_threading():
+    """Set the numba threading layer.
+
+    For parallel numba jit blocks, the backend threading layer is selected at runtime
+    based on an order set inside the numba package.  We would like to change the
+    order of selection to prefer the OpenMP backend in order to be compatible with
+    other pieces of compiled code that use the OpenMP thread pool.  If OpenMP is not
+    supported, then we next try to use TBB for threading and finally fall back to the
+    default, which uses a multiprocessing workqueue.
+
+    Args:
+        None
+
+    Returns:
+        None
+
+    """
+    log = Logger.get()
+    threading = "default"
+    try:
+        from numba.npyufunc import omppool
+
+        threading = "omp"
+    except ImportError:
+        # no OpenMP support
+        try:
+            from numba.npyufunc import tbbpool
+
+            threading = "tbb"
+        except ImportError:
+            # no TBB
+            pass
+    try:
+        from numba import config, njit, threading_layer
+
+        config.THREADING_LAYER = threading
+
+        # In order to get numba to actually select a threading layer, we must
+        # trigger compilation of a parallel function.
+        @njit(parallel=True)
```

Comment:
This might help, `@vectorize(target='parallel')` is supported on more platforms than `@njit(parallel=True)`:
```python
import numba
numba.config.THEADING_LAYER='openmp'
from numba import vectorize
import numpy as np

@vectorize('float64(float64)', target='parallel')
def force_thread_launch(x):
    return x + 1

force_thread_launch(np.zeros(1))

print(numba.threading_layer())
```
it also feels like finding out what threading layer will be used should be easier... if that would be useful please open a feature request on the Numba issue tracker.